### PR TITLE
refactor: aws deployments with new infra 

### PR DESCRIPTION
**Pull request by iadjadj**

### CODE REVIEW

File: infrastructure/quick-deploy/aws/multi-stages/armonik/locals.tf [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  job_partitions_in_database_image  = "${var.aws_account_id}.dkr.ecr.${var.region}.amazonaws.com/${var.suffix}/${var.job_partitions_in_database.image}"
+  control_plane_image               = "${var.aws_account_id}.dkr.ecr.${var.region}.amazonaws.com/${var.suffix}/${var.control_plane.image}"
+  admin_old_gui_api_image           = "${var.aws_account_id}.dkr.ecr.${var.region}.amazonaws.com/${var.suffix}/${var.admin_old_gui.api.image}"
+  admin_old_gui_old_image           = "${var.aws_account_id}.dkr.ecr.${var.region}.amazonaws.com/${var.suffix}/${var.admin_old_gui.old.image}"
+  ingress_image                     = "${var.aws_account_id}.dkr.ecr.${var.region}.amazonaws.com/${var.suffix}/${var.ingress.image}"
+  authentication_image              = "${var.aws_account_id}.dkr.ecr.${var.region}.amazonaws.com/${var.suffix}/${var.authentication.image}"
+  compute_plane_polling_agent_image = "${var.aws_account_id}.dkr.ecr.${var.region}.amazonaws.com/${var.suffix}/${var.compute_plane.default.polling_agent.image}"
+  compute_plane_worker_image        = "${var.aws_account_id}.dkr.ecr.${var.region}.amazonaws.com/${var.suffix}/${var.compute_plane.default.worker[0].image}"
```
Comment (lzianekhodja) [inline infrastructure/quick-deploy/aws/multi-stages/armonik/locals.tf:11]:
```suggestion
  compute_plane_worker_image        = { for key, value in var.compute_plane: key => [for w in value.worker: "${var.aws_account_id}.dkr.ecr.${var.region}.amazonaws.com/${var.suffix}/${w.image}"] }
```

File: infrastructure/quick-deploy/aws/multi-stages/armonik/main.tf [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      priority              = 1
+      reserved_pods         = 1
+      max_pods              = 100
+      preemption_percentage = 50
+      parent_partition_ids  = []
+      pod_configuration     = null
+    }
+    # Update images for polling_agent et worker
+    polling_agent = merge(v.polling_agent, { image = local.compute_plane_polling_agent_image })
+    worker        = [for w in v.worker : merge(w, { image = local.compute_plane_worker_image })]
```
Comment (lzianekhodja) [inline infrastructure/quick-deploy/aws/multi-stages/armonik/main.tf:24]:
```suggestion
    worker        = [for w in v.worker : merge(w, { image = local.compute_plane_worker_image[k][index(v.worker, w)] })]
```

File: infrastructure/quick-deploy/aws/multi-stages/armonik/parameters.tfvars [hunk 3]
```diff
@@ -145,7 +151,184 @@ compute_plane = {
...
+        cpu    = "1000m"
+        memory = "256Mi"
+      }
+    }
+    # ArmoniK workers
+    worker = [
+      {
+        name              = "worker"
+        image             = "armonik-bench-worker"
+        tag               = "0.12.1"
```
Comment (lzianekhodja) [inline infrastructure/quick-deploy/aws/multi-stages/armonik/parameters.tfvars:289]:
```suggestion
        tag               = "0.14.3"
```

File: infrastructure/quick-deploy/aws/multi-stages/armonik/parameters.tfvars [hunk 4]
```diff
@@ -161,7 +344,7 @@ compute_plane = {
     worker = [
       {
         name              = "worker"
-        image             = "125796369274.dkr.ecr.eu-west-3.amazonaws.com/armonik-worker"
+        image             = "armonik-stream-worker"
         tag               = "0.12.1"
```
Comment (lzianekhodja) [inline infrastructure/quick-deploy/aws/multi-stages/armonik/parameters.tfvars:348]:
```suggestion
        tag               = "0.14.3"
```

**Review by lemaitre-aneo — COMMENTED** [review]
You should remove the previous `all` files. That would allow git to see that files were moved.

**Review by lzianekhodja — APPROVED** [review]

**Pull request merged**